Simplevars.entry allows a space between the key and the value,
sysctl(8) doesn't take that as a valid entry.

--- lenses/sysctl.aug.orig	Tue Jun  2 00:03:07 2015
+++ lenses/sysctl.aug	Tue Jun  2 10:11:04 2015
@@ -33,8 +33,12 @@ let filter = incl "/boot/loader.conf"
 (* View: comment *)
 let comment = Util.comment_generic /[ \t]*[#;][ \t]*/ "# "
 
+(* View: entry *)
+let entry = Build.key_value Rx.word Sep.equal
+            (store Simplevars.to_comment_re . (Util.eol|Util.comment_eol))
+
 (* View: lns
      The sysctl lens *)
-let lns = (Util.empty | comment | Simplevars.entry)*
+let lns = (Util.empty | comment | entry)*
 
 let xfm = transform lns filter
